tests/unit/SpecialResolutionForm.spec.ts

```diff
@@ -0,0 +1,102 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+
+import { getVuexStore } from '@/store/'
+import { createLocalVue, mount } from '@vue/test-utils'
+import SpecialResolutionForm from '@/components/SpecialResolution/SpecialResolutionForm.vue'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+import { HelpSection } from '@/components/common/'
+import { EmptySigningPersonIF } from '@/interfaces'
+import { CooperativeResource } from '@/resources/SpecialResolution/CooperativeResource'
+
+Vue.use(Vuetify)
+
+const localVue = createLocalVue()
+const vuetify = new Vuetify({})
+
+describe('Special Resolution Form component', () => {
+  let wrapper: any
+  let store: any = getVuexStore()
+
+  const entitySnapshot = {
+    businessInfo: {
+      legalName: 'Mock Original Name',
+      legalType: 'CP'
+    }
+  }
+
+  beforeAll(() => {
+    // init store
+    store.state.stateModel.currentJsDate = new Date('2020-03-01T16:30:00Z')
+    store.state.stateModel.tombstone.currentDate = '2021-03-01'
+    store.state.stateModel.entitySnapshot = entitySnapshot
+    store.state.stateModel.createResolution = {
+      resolutionText: '',
+      signingPerson: { ...EmptySigningPersonIF },
```

Comment:
It's better to hard-code data here instead of using IFs or objects, so that 2 mistakes don't cancel each other out.